includes/razorpay-webhook.php

```diff
@@ -96,6 +96,13 @@ public function process()
             return;
         }
 
+        // Skip the webhook if not the valid data and event
+        if ($this->shouldConsumeWebhook($data) === false) {
+
+            rzpLogDebug("Webhook process exited in shouldConsumeWebhook function");
```

Comment:
@rohitcbr please change message and log data as you are using  rzpLogDebug

Reply:
@abdulwahidsharief updated here https://github.com/razorpay/razorpay-woocommerce/pull/579/commits/0ee45d4010556d22d68a9a8a0f263b6a38c8d8b6